accounts-bench/src/main.rs

```diff
@@ -0,0 +1,103 @@
+use clap::{value_t, value_t_or_exit, App, Arg};
+use rayon::prelude::*;
+use solana_measure::measure::Measure;
+use solana_runtime::accounts::{create_test_accounts, update_accounts, Accounts};
+use solana_sdk::pubkey::Pubkey;
+use std::collections::HashMap;
+use std::fs;
+use std::path::PathBuf;
+
+fn main() {
+    solana_logger::setup();
+
+    let matches = App::new("crate")
+        .about("about")
+        .version("version")
+        .arg(
+            Arg::with_name("num_slots")
+                .long("num_slots")
+                .takes_value(true)
+                .value_name("SLOTS")
```

Comment:
How about memorizing your favorite number as the default? :) Or, there is none?

Reply:
yea. I'll add a default.